modyn/model_storage/model_storage.py

```diff
@@ -1,34 +1,31 @@
 import os
 import pathlib
-from typing import Optional, Tuple
+import shutil
+import tempfile
 
-from jsonschema import ValidationError
 from modyn.common.ftp.ftp_server import FTPServer
 from modyn.model_storage.internal.grpc.grpc_server import GRPCServer
-from modyn.utils import validate_yaml
 
 
 class ModelStorage:
     def __init__(self, config: dict) -> None:
         self.config = config
+        self._setup_model_storage_directories()
 
-        valid, errors = self._validate_config()
-        if not valid:
-            raise ValueError(f"Invalid configuration: {errors}")
+    def _setup_model_storage_directories(self) -> None:
+        self.model_storage_directory = pathlib.Path(os.getcwd()) / "model_storage"
+        self.ftp_directory = pathlib.Path(tempfile.gettempdir()) / "ftp_model_storage"
```

Comment:
I see. Yes, the ftp directory is temporary. It temporarily stores the models that are served to other components. I decided to put it in the cwd. In contrast, I made the model storage directory (containing the stored models) configurable. It is now a separate volume and attached to the model storage service.

Reply:
Hmm, I think the CWD is not the best choice here, sorry. I think this should also be configurable. Sorry 'bout that, but e.g. on gcloud this would be problematic

Reply:
Okay, no problem. Now, we use 2 subdirectories of the configurable model storage directory. The first one is used to store the trained models  and the second is designated to the FTP server to serve the loaded models.

Reply:
Merci!